core/server/data/migrations/versions/3.0/2-populate-type-column.js

```diff
@@ -0,0 +1,94 @@
+const Promise = require('bluebird');
+const toPairs = require('lodash/toPairs');
+const common = require('../../../../lib/common');
+
+/*
+ * @param from: object with a SINGLE entry { 'fromColumn': 'fromValue' }
+ * @param to: object with a SINGLE entry { 'toColumn': 'toValue' }
+ */
+const createColumnToColumnMap = ({from, to, tableName}) => (connection) => {
+    return connection.schema.hasTable(tableName)
+        .then((tableExists) => {
+            if (!tableExists) {
+                common.logging.warn(
+                    `Table ${tableName} does not exist`
+                );
+                return;
+            }
+
+            const [fromColumn, fromValue] = toPairs(from)[0];
+            const [toColumn, toValue] = toPairs(to)[0];
+
+            return Promise.all([
+                connection.schema.hasColumn(tableName, fromColumn),
+                connection.schema.hasColumn(tableName, toColumn)
+            ]).then(([fromColumnExists, toColumnExists]) => {
+                if (!fromColumnExists) {
+                    common.logging.warn(
+                        `Table '${tableName}' does not have column '${fromColumn}'`
+                    );
+                }
+                if (!toColumnExists) {
+                    common.logging.warn(
+                        `Table '${tableName}' does not have column '${toColumn}'`
+                    );
+                }
+                if (!fromColumnExists || !toColumnExists) {
+                    return;
+                }
+
+                common.logging.info(
+                    `Updating ${tableName}, setting ${fromValue} in ${fromColumn} to ${toValue} in ${toColumn}`
```

Comment:
This log message is confusing. Currently, it produces this output:
```
Updating posts, setting true in page to page in type
```
 
Maybe something following would make it clearer:
```
`Updating ${tableName}, setting "${toValue}" value in "${toColumn}" column from "${fromValue}" value in "${fromColumn}" column`
```
Then an example output would read:
```
Updating posts, setting "page" value in "type" column from  "true" value in "page" column
```

Reply:
I've opted for something which would produce output like 

```
Updating posts, setting "type" column to "page" where "page" column is "true"
```